src/translator/byte_array_util.h

```diff
@@ -7,6 +7,9 @@ namespace bergamot {
 AlignedMemory loadFileToMemory(const std::string& path, size_t alignment);
 AlignedMemory getModelMemoryFromConfig(marian::Ptr<marian::Options> options);
 AlignedMemory getShortlistMemoryFromConfig(marian::Ptr<marian::Options> options);
+void getVocabsMemoryFromConfig(marian::Ptr<marian::Options> options,
+                               std::vector<AlignedMemory>& vocabsMemory,
+                               std::vector<string_view>& vocabsMemorySV);
```

Comment:
This should be `AlignedMemory`, not `string_view`. Reasons 1) Consistency, API person outside shouldn't work with 2 structures doing same thing. 2) `AlignedMemory` + WASM glue has been worked out already over [here](https://github.com/browsermt/bergamot-translator/blob/2788116f8b10f7edd2168ae7fb0be0f059681c8b/wasm/bindings/TranslationModelBindings.cpp#L13-L23).

/cc @abhi-agg